docs/cromwell_features/RetryWithMoreMemory.md

```diff
@@ -0,0 +1,35 @@
+#Retry with More Memory
+
+With this feature one can specify an array of strings which when encountered in the `stderr` file by Cromwell, 
+allows the task to be retried with more memory. The retry will be counted against the `maxRetries` count mentioned in 
+the `runtimeAtrributes` in the task. There are 2 settings for this feature:
+* `system.memory-retry-error-keys` : the error keys that need to be set in Cromwell config
+* `memory_retry_multiplier` : [optional] the factor by which the memory should be multiplied while retrying. This needs 
+to be passed in through workflow options and should be in the range `1.0 <= multiplier <= 99.0` (note: if set to `1.0` the task
+will retry with same amount of memory). If not passed, Cromwell will not retry the task with more memory.
```

Comment:
Could you clarify whether this means:
* "will not retry the task" 
* "will retry the task but with the same amount of memory"?